packages/web/src/app/doc-pages/brand/color-doc/color-picker/color-picker.component.html

```diff
@@ -1,20 +1,14 @@
 <div
   class="theme-container e-color-background-1"
-  [ngClass]="theme === 'dark' ? 'e-theme-dark' : 'e-theme-light'"
+  [ngClass]="theme() === 'dark' ? 'e-theme-dark' : 'e-theme-light'"
 >
   <div class="header-container">
-    <app-color-picker-header (changeThemeEvent)="handleChangeThemeEvent($event)" [currentTheme]="theme" />
+    <app-color-picker-header />
   </div>
-  <app-color-picker-exhibit [currentColor]="currentColor" [theme]="theme" [colorList]="colorList" />
+  <app-color-picker-exhibit [theme]="theme" />
```

Comment:
Is `theme` supposed to be here?
```suggestion
  <app-color-picker-exhibit />
```